api/db/changelog/db.changelog-166-add-access-module-tables.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-166-add-access-module-tables">
+    <createTable tableName="access_module">
+      <column name="name" type="ENUM('ERA_COMMONS', 'TWO_FACTOR_AUTH', 'RAS_LOGIN_GOV', 'RT_COMPLIANCE_TRAINING', 'DATA_USER_CODE_OF_CONDUCT', 'PROFILE_CONFIRMATION', 'PUBLICATION_CONFIRMATION')">
```

Comment:
Cool - thanks for the experimentation here.

If we need to add a new entry here, what would the Liquibase change look like? Also - how does this manifest in the Hibernate model?

Reply:
Upgrade:  Liquibase would be just the same in `<sql>` section below.
Hibernate: Found some online intro on how to do it, so I did not try it myself. That would be my next PR's task. And even I am not sure if we should start considering using JDBC or continue use Hibernate

Reply:
RE Liquibase: I don't understand, I get the insertions of new modules (via `<sql>`), but how would this enumerated type list get extended with the new value? Does this require a column update statement?

Reply:
Sorry, wrong command, this should be the one in below comment: 
`mysql> alter TABLE access_module MODIFY name ENUM('RT_COMPLIANCE_TRAINING', 'DATA_USER_CODE_OF_CONDUCT', 'RAS_LOGIN_GOV', 'RAS_LOGIN_GOV_2');`